grammars/sql.cson

```diff
@@ -106,7 +106,7 @@
     'name': 'storage.modifier.sql'
   }
   {
-    'match': '\\b\\d+\\b'
+    'match': '\\b\\d+(\\.\\d+)?\\b'
```

Comment:
So I did a little test:

```sql
mysql> INSERT INTO NumberTest (number) VALUES (1.0);
mysql> INSERT INTO NumberTest (number) VALUES (1.);
mysql> INSERT INTO NumberTest (number) VALUES (.1);
mysql> SELECT * FROM NumberTest;
+----+--------+
| id | number |
+----+--------+
|  1 | 1.0000 |
|  2 | 1.0000 |
|  3 | 0.1000 |
+----+--------+

mysql> SELECT * FROM NumberTest WHERE number=.1;
+----+--------+
| id | number |
+----+--------+
|  3 | 0.1000 |
+----+--------+

mysql> SELECT * FROM NumberTest WHERE number=1.;
+----+--------+
| id | number |
+----+--------+
|  1 | 1.0000 |
|  2 | 1.0000 |
+----+--------+
```

And numbers can indeed start or end with a period.

Should I use `(\\d+)|(\\.\\d+)|(\\d+\\.)|(\\d+\\.\\d+)` then, or is there a better way?